Dockerfile.modules

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
```

Comment:
`MAINTAINER` is deprecated and been replaced with `LABEL`... But we're not publishing this image so may as well just remove.